stack-release.html

```diff
@@ -95,12 +96,13 @@
 							compareDiffAndUpdateElement(
 								"src_latest_tag_vs_main",
 								"synapse-web-monorepo",
+								synapseReactClientLatestVersionTag,
 								"main",
```

Comment:
Use the correct tag name and swap the order so the tag branch is the base for this diff (because we want to see if the tag is missing anything from `main`)